lib/js/importers/configs/allInOneConfig.json

```diff
@@ -0,0 +1,48 @@
+{
+  "destinationPath": "lib/styles/settings/colors/",
+  "bins": [
+    {
+      "id": "620e4f834bf50f4b2d0044e9",
+      "name": "wnl",
+      "isTheme": false,
```

Comment:
is it used?

Reply:
It was used in previous importers. I thought also it will be use in views, but views have manually defined loaded files. So I will remove it, but I suppose we can back to this option in the future.